cdap-app-fabric/src/main/java/io/cdap/cdap/internal/tether/TetherRequestBase.java

```diff
@@ -0,0 +1,32 @@
+/*
+ * Copyright © 2021 Cask Data, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not
+ * use this file except in compliance with the License. You may obtain a copy of
+ * the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations under
+ * the License.
+ */
+
+package io.cdap.cdap.internal.tether;
+
+/**
+ * Base class for client and server tether requests.
+ */
+public class TetherRequestBase {
```

Comment:
Do we plan to add more details here? If not, I think we dont need this class.

Reply:
Removed this class